server/controllers/feedback.js

```diff
@@ -0,0 +1,88 @@
+// connect to the Feedback Model
+const Feedback = require("../models/feedback");
+const Interview = require("../models/interview");
+
+module.exports.createFeedback = (req, res, next) => {
+  const newFeedback = new Feedback({
+    interview: req.body.interview,
```

Comment:
suggestion to refactor these few lines because there's a lot of repeated code here. 2 options here:

store `req.body` in a variable so you don't need to keep repeating it :) 

second option, because majority, if not all, the keys names are the same, you can deconstruct the object. (even if the keys are not the same, you can rename them) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring

so it would look something like this: 
```
const {interview, candidate, overallScore, communication, codeEfficiency, ...} = req.body;
const newFeedback = new Feedback({interview, candidate, overallScore, communication, ...});
```

Reply:
The request body has been destructured as requested